src/containers/TopNavBar.vue

```diff
@@ -0,0 +1,17 @@
+<template>
+  <header
+    class="relative flex h-16 w-full flex-col items-center justify-between"
+  >
+    <div
+      class="flex h-16 w-full items-center justify-between border-gray_02-light p-4 dark:border-gray_02-dark"
+    >
+      <Logo />
+      <ThemeSwitcher />
+    </div>
+  </header>
+</template>
+
+<script setup lang="ts">
+import Logo from '@src/components/Logo.vue';
+import ThemeSwitcher from '@src/components/ThemeSwitcher.vue';
```

Comment:
```suggestion
import Logo from '@components/Logo.vue';
import ThemeSwitcher from '@components/ThemeSwitcher.vue';
```

나중에 구조분해할당하여 사용하는 방법도 알려드릴게요. 아래는 예시입니다.
```suggestion
import { Logo, ThemeSwitcher } from '@components';
```